### ISSUE TITLE
Please join my group 

### ISSUE THREAD

**Issue by javascript-programmer** [Accio1/Accio1#1]
Hi There Could You Please join My team
 Cuz I joined the Nasa space app challenge
and They Said I need To have a team
Cuz I need you To so I can be Able to join 
Otherwise I won't Be accepted

**Comment by Accio1** [comment]
@javascript-programmer Could you give a bit more information?

**Comment by javascript-programmer** [comment]
Yes , Well I hosted a project/responsntirey For The Nasa Space App challenge and they said I need 
To Have A Team So I can join Else I will not be able to join 
Well The Project Will be called spacelearn That Will Give People Facts about planets when you click on them**
I Will Give You What They Said Each Local Lead is responsible for their own partner outreach efforts. You may want to consider inviting local organizations to be a part of the Space Apps Challenge event, including software and technology companies, telecommunications companies, science organizations, non-profits, civil society organizations, government offices, universities, etc.  In addition, while we are hosting virtually, there is still the opportunity to have partners and sponsors provide local awards, marketing, and other kids of support ( You Might Not Understand This)

**Comment by Accio1** [comment]
@javascript-programmer I don't see a repository on your profile that seems to be related to this. What specifically do you need my help with?

**Comment by javascript-programmer** [comment]
I still didn't make it because its starting at october 1 lol! Also i need help with the css
